packages/kbn-optimizer/limits.yml

```diff
@@ -41,7 +41,7 @@ pageLoadAssetSize:
   monitoring: 80000
   navigation: 37269
   newsfeed: 42228
-  observability: 89709
+  observability: 95000
```

Comment:
@spalger Yep I tried quite a few things as you can see in my commit history https://github.com/elastic/kibana/pull/124132/commits starting from commit `627a6265cf2b28ff212a267b923b362a4d0b1952`

1. I used await import for the import suggested by @simianhacker https://github.com/elastic/kibana/pull/124132/commits/627a6265cf2b28ff212a267b923b362a4d0b1952 -> This didn't solve the Metric CI error (the bundle size increased even more) plus this change  ended up in a few typescript errors that had to be fixed. Since Metrics CI error was not fixed by the await import, I didn't solve the typescript errors and continued with the next experiment
2. I used await import for another new import that was introduced in this PR https://github.com/elastic/kibana/pull/124132/commits/05c170c36290c0f44906c3bb1495bebf599af187. Again Metrics error persisted in the CI
3. I tried a few more experiments where I removed pretty much everything from the new Observability page that is introduced in this PR. This again didn't solve the problem
4. That's when I started looking into the numbers reported from the CI and I concluded that it should be the limit that should be changed. It is a brand new page, so it made sense to me that the limit should be increased.  

I gave it a try to run `node scripts/build_kibana_platform_plugins --focus observability --profile`, but I didn't get any `stats.json` generated. At this point I didn't investigate further why and where should I find the stats, cause I was pretty much sure it is a limit issue. 

@simianhacker Can you give your thoughts on this?